index.html

```diff
@@ -9,51 +9,229 @@
+                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <button type="button" class="get-resume">Get my resume</button>
+
+    <!--Languages-Skills Section-->
+    <section class="skills-lang">
         <ul>
-            <li>
-                <a href="#" class="social twiter">
-                    <img src="./images/twiter.png" alt="Twitter"/>
-                </a>
+            <li id="languages">
+                <a href="#" class="languages">
+                    <span class="lang-head">Languages</span> <img src="./images/Chevron-Down.png" alt="Chevron-Down"/>
+                </a> 
             </li>
-            <li>
-                <a href="#" class="social linkendIn">
-                    <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
-                </a>
+            <li class="about-lang">
+                <img src="./images/javaScript-icon.png" alt="JavaScript"/>
+                <a href="#" class="lang javascript">JavaScript</a> 
             </li>
-            <li>
-                <a href="#" class="social twiter">
-                    <img src="./images/mail.png" alt="Email"/>
-                </a>
+            <li class="about-lang">
+                <img src="./images/html-icon.png" alt="Html"/>
+                <a href="#" class="lang html">HTML</a> 
             </li>
-            <li>
-                <a href="#" class="social github">
-                    <img src="./images/Vector.png" alt="GitHub"/>
-                </a>
+            <li class="about-lang">
+                <img src="./images/css-icon.png" alt="CSS"/>
+                <a href="#" class="lang css">CSS</a> 
             </li>
-            <li>
-                <a href="#" class="social peace">
-                    <img src="./images/fingers.png" alt="Peace"/>
-                </a>
+            <li class="framework-skills">
+                <a href="#" class="framework">Framework</a>
+                <img src="./images/Chevron-Right.png" alt="Right-caret"/>
+            </li>
+            <li class="framework-skills">
+                <a href="#" class="skills">Skills</a>
+                <img src="./images/Chevron-Right.png" alt="Right-caret"/>
             </li>
         </ul>
     </section>
+    </section>
```

Comment:
Your technology categories section is not in accordance with the Figma template of your choosing. Please update this accordingly.
| Figma template  | Your design |
| ------------- | ------------- |
| ![langs](https://user-images.githubusercontent.com/33268927/170699468-19d0454b-d7aa-4866-8734-65a114110a27.png)| ![lang-bad](https://user-images.githubusercontent.com/33268927/170699492-5aded051-037e-43ed-93df-0b84a85b03a6.png) |